### ISSUE TITLE
Error: Unknown playlist type: <PlaylistType.PLACEHOLDER: 3>

### ISSUE THREAD

**Issue by doofusdavid** [rckclmbr/pyportify#25]
Hey there.

This is in OSX, Yosemite.  Trying to move my playlists off, and ran into this error.  It results in a blank playlist page, so I can't transfer anything over.  I don't see anything unusual in my playlists.  There are long playlists, folders containing playlists, quite a few playlists, but I don't really see anything that could be construed as a placeholder.  Any thoughts?

```
➜  ~  pyportify
Open your browser and go to http://localhost:3132
Managing socket.io
Socketio session started
New connection
127.0.0.1 - - [2015-05-08 13:36:40] "POST /google/login HTTP/1.1" 200 162 0.188106
127.0.0.1 - - [2015-05-08 13:36:48] "POST /spotify/login HTTP/1.1" 200 162 0.376064
2015-05-08 13:36:48,698 - pyportify.views - ERROR - Exception on /spotify/playlists [GET]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Library/Python/2.7/site-packages/pyportify/views.py", line 142, in spotify_playlists
    ret_playlists = fetch_spotify_playlists()
  File "/Library/Python/2.7/site-packages/pyportify/views.py", line 156, in fetch_spotify_playlists
    for playlist in playlists:
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/_abcoll.py", line 581, in __iter__
    v = self[i]
  File "/Library/Python/2.7/site-packages/spotify/__init__.py", line 60, in wrapper
    return f(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/spotify/playlist_container.py", line 193, in __getitem__
    raise spotify.Error('Unknown playlist type: %r' % playlist_type)
Error: Unknown playlist type: <PlaylistType.PLACEHOLDER: 3>
127.0.0.1 - - [2015-05-08 13:36:48] "GET /spotify/playlists HTTP/1.1" 500 412 0.038552
```

**Comment by rckclmbr** [comment]
Do you have any idea what a "Placeholder" playlist would be?  I'm... actually not sure.

Regardless, it's a documented type in the API, I'll file a ticket with pyspotify.

**Comment by doofusdavid** [comment]
Best guess: https://community.spotify.com/t5/Closed-Ideas/Spacers-For-Playlists/idi-p/995648

They mention being able to create a blank spacer playlist by just calling it '-'.  I believe I did this at some point, and decided it wasn't helping, and deleted it.  It also looks like it's no longer supported in current versions of the desktop app, so maybe I'm wrong and it's something else entirely.

**Comment by rckclmbr** [comment]
This should be fixed by 44af40d5accb32f72d85b7221b36ee00db9839a0

Run `pip install --upgrade pyportify` to get the latest version, let me know how it works for you.

**Comment by doofusdavid** [comment]
Like a champ!  Thanks so much!
